flocker/node/test/test_deploy.py

```diff
@@ -703,6 +703,32 @@ def test_does_not_exist(self):
 DISCOVERED_APPLICATION_WITH_VOLUME = APPLICATION_WITH_VOLUME
 
 
+class DeployerDiscoverStateTests(SynchronousTestCase):
```

Comment:
Not necessary if you have an appropriate test in `testools` generic IDeployer test factory - see comment there.